Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM --platform=$BUILDPLATFORM rust:1 AS builder
```

Comment:
I don't understand why this dockerfile duplicates the build logic. We already build DEB/RPM packages for both x86_64 and arm64 (aarch64) targets. Why the separate build workflow to maintain?

Reply:
This is not a seperate workflow, but rather a way to build for multiarch images using buildx. I could not find a better way of doing this.

Reply:
This is a list of dependencies and build requirements, isn't it? Can the docker leverage the DEBs we already provide? I don't like the idea of maintaining the list of build dependencies in the Dockerfile in addition to the package control. It feels like this could be done differently.